src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -11,7 +12,8 @@
      * Вернуть сумму нечетных числел или 0, если таких несуществует</p>
      **/
     public Integer oddSum(List<Integer> numbers) {
-        return null;
+        return numbers.stream().filter(number -> number % 2 == 1)
+                .mapToInt(Integer::intValue).sum();
```

Comment:
do we need to map int to int?

Reply:
I guess only if we want to use sum(), or I can just use reduce()